Program.cs

```diff
@@ -545,6 +553,49 @@ public void CopyValidationReport()
       }
     }
 
+    public void CopyValidationReportMarkdown()
+    {
+      List<MarkdownIssue> ConvertToMarkdown(List<Issue> rawIssues)
+      {
+        var dotnetIssues = new List<MarkdownIssue> { };

```

Comment:
`dotnteIssues` should be `markdownIssues` I guess?

Reply:
Feedback applied